cranelift/codegen/src/isa/aarch64/lower.isle

```diff
@@ -1371,68 +1371,55 @@
       (value_regs_get src 0))
 
 
-;;;; Rules for `fcmp` 32 bit ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;; Rules for `fcmp` ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-(rule (lower (has_type ty @ (multi_lane _ _) (fcmp (fcmp_zero_cond_not_eq cond) x (splat (f32const (zero_value_f32 y))))))
+(rule (lower (has_type ty @ (multi_lane _ _) (fcmp (fcmp_zero_cond_not_eq cond) x y)))
+      (if-let _ (zero_value y))
```

Comment:
`(if-let _ ...)` has a short form `(if ...)` that you could use here! And likewise in a few spots below.

Reply:
Ah, nice.